.github/workflows/release-dataset.yml

```diff
@@ -0,0 +1,72 @@
+name: Publish dataset to production DB
+
+on:
+  workflow_dispatch:
+    branches: 
+      - main
+    inputs: 
+      megaset-name:
+        description: "Enter the name of the megaset to release every dataset within"
+        required: false
+      contained-dataset-id: 
+        description: "Enter the ID in the format [NAME]_v[VERSION] to release a single dataset contained within a megaset"
+        required: false
+      standalone-dataset-id: 
```

Comment:
I don't think you need both `contained-dataset-id` and `standalone-dataset-id`. You could just have `dataset-id`.